addons/mass_mailing/models/mailing.py

```diff
@@ -102,7 +102,7 @@ def _get_default_mail_server_id(self):
     mail_server_id = fields.Many2one('ir.mail_server', string='Mail Server',
         default=_get_default_mail_server_id,
         help="Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.")
-    contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists')
+    contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists', default=lambda self: self.env['mailing.list'].search([]) if len(self.env['mailing.list'].search([])) == 1 else False)
```

Comment:
is that really needed ? -> that the thing that makes the query count increases I guess
And increasing query count is not cool, even if sometimes, necessary. sooo.. really needed ?
If yes, I would squash the query count update commit into this one.. Or put everything that prepare the tour into in a same commit, than the tour itself in another one.
You choose.

Reply:
Note that this field is only used if mailing model is 'mailing.list'.
So if you do so, if there is only one mailing.list, for each mailing created, even if it's targeting another model, it will set contact_list_ids. If it's really needed, I would also check the mailong model before setting the default value.

Reply:
what happens if you do self.env['mailing.mailing'].create({'mailing_model_id': crm_lead_mode_id}) ?
your contact list will be set to mailing_list_1. Which makes no sense.
Extract you default into a small default method before fields declaration and check if model_id is mailing_list.